### DISCUSSION TITLE
请问有没有悬浮在搜索结果上可预览播放的组件？

### DISCUSSION THREAD

**Discussion by winter60** [the1812/Bilibili-Evolved#4819] [提问 / Q&A]
请问有没有悬浮在搜索结果上可预览并播放的组件呀， 可播放，可拖动进度
![image](https://github.com/the1812/Bilibili-Evolved/assets/23653326/37b17324-eaf2-452f-8549-e17c5a8ed4d9)

**Comment by the1812** [comment]
鼠标悬浮一会就有, 只是不能拖进度

**Comment by Geeyun-JY3** [comment]
关联：#4716
